### ISSUE TITLE
config.location containing a newline at EOF will crash MaxTo

### ISSUE THREAD

**Issue by r15ch13** [digitalcreations/MaxTo#454]
**Describe the bug**
Some text editors are set to always add a newline at the EOF. MaxTo crashes if the `config.location` contains an additional newline, so creating or changing the file will crash it on start.

**To Reproduce**
1. Create `config.location` file
2. Add path and additional newline at the EOF. It should look like this:
> ```
>  .\maxto
>
>  ```
3. Start MaxTo

**Expected behavior**
Remove additonal whitespaces from `config.location` content via `.Trim()`

**Screenshots**
![image](https://user-images.githubusercontent.com/432127/62946882-d5934300-bde1-11e9-857d-19510c4c68ca.png)

**System information:**
 - Windows version: 10
 - MaxTo version: 2.0.1

**Additional context**
```log
2019-08-13 15:43:36 [Core@2.0.1.499] [Information] Logger setup (online logging: True, process #2444)
2019-08-13 15:43:36 [Core@2.0.1.499] [Information] Using configuration path ".\maxto
"
2019-08-13 15:43:36 [Core@2.0.1.499] [Fatal] Exception occured during startup
System.Reflection.TargetInvocationException: Exception has been thrown by the target of an invocation. ---> System.ComponentModel.Win32Exception: The filename, directory name, or volume label syntax is incorrect
   at MaxTo.Core.PathHelper.GetDiskSpace(String path)
   at MaxTo.Core.Settings.FileBasedConfigurationStore..ctor(ILogger logger, IConfigurationLocation location)
   --- End of inner exception stack trace ---
   at System.RuntimeMethodHandle.InvokeMethod(Object target, Object[] arguments, Signature sig, Boolean constructor)
   at System.Reflection.RuntimeConstructorInfo.Invoke(BindingFlags invokeAttr, Binder binder, Object[] parameters, CultureInfo culture)
   at System.RuntimeType.CreateInstanceImpl(BindingFlags bindingAttr, Binder binder, Object[] args, CultureInfo culture, Object[] activationAttributes, StackCrawlMark& stackMark)
   at System.Activator.CreateInstance(Type type, BindingFlags bindingAttr, Binder binder, Object[] args, CultureInfo culture, Object[] activationAttributes)
   at System.Activator.CreateInstance(Type type, Object[] args)
   at Caliburn.Micro.SimpleContainer.ActivateInstance(Type type, Object[] args)
   at Caliburn.Micro.SimpleContainer.<>c__DisplayClass7_0.<RegisterSingleton>b__0(SimpleContainer container)
   at System.Linq.Enumerable.WhereSelectArrayIterator`2.MoveNext()
   at System.Collections.Generic.List`1.InsertRange(Int32 index, IEnumerable`1 collection)
   at Caliburn.Micro.SimpleContainer.DetermineConstructorArgs(Type implementation)
   at Caliburn.Micro.SimpleContainer.<>c__DisplayClass7_0.<RegisterSingleton>b__0(SimpleContainer container)
   at Caliburn.Micro.ContainerExtensions.GetInstance[TService](SimpleContainer container, String key)
   at MaxTo.Core.IoC.ContainerExtensions.<>c__1`2.<ReregisterAs>b__1_0(SimpleContainer c)
   at System.Linq.Enumerable.WhereSelectListIterator`2.MoveNext()
   at System.Collections.Generic.List`1..ctor(IEnumerable`1 collection)
   at System.Linq.Enumerable.ToList[TSource](IEnumerable`1 source)
   at Caliburn.Micro.SimpleContainer.GetInstance(Type service, String key)
   at System.Linq.Enumerable.WhereSelectArrayIterator`2.MoveNext()
   at System.Collections.Generic.List`1.InsertRange(Int32 index, IEnumerable`1 collection)
   at Caliburn.Micro.SimpleContainer.DetermineConstructorArgs(Type implementation)
   at Caliburn.Micro.SimpleContainer.<>c__DisplayClass7_0.<RegisterSingleton>b__0(SimpleContainer container)
   at Caliburn.Micro.ContainerExtensions.GetInstance[TService](SimpleContainer container, String key)
   at MaxTo.Core.MainForm.<OnLoad>d__20.MoveNext()
```

**Comment by vegardlarsen** [comment]
Thank you for the report. I can see how that would be annoying, I will fix this for version 2.1.

**Comment by vegardlarsen** [comment]
I have fixed this issue and it will be in version 2.1.
